examples/platforms/utils/soft_source_match_table.c

```diff
@@ -212,7 +212,7 @@ static inline void addToSrcMatchExtIndirect(uint8_t entry, const otExtAddress *a
     srcMatchExtEntry[entry].allocated = true;
 }
 
-static inline void removeFromSrcMatchExtIndirect(uint8_t entry)
+static inline void removeFromSrcMatchExtIndirect(uint16_t entry)
```

Comment:
remove inline?

Reply:
The [page](https://www.geeksforgeeks.org/inline-functions-cpp/) lists the cases that can't use `inline` keyword. This function doesn't match any item of the lists. I think it is ok to use `inline` in this function.